deploy/helm/events-collector-config.yaml

```diff
@@ -311,7 +370,7 @@ service:
       receivers:
         - k8s_events
 {{- end }}
-{{- if .Values.otel.manifests.enabled }}
+{{- if and .Values.otel.events.enabled .Values.otel.manifests.enabled }}
```

Comment:
This condition makes no sense anymore, whole events deployment has no condition and we want this pipeline to run even if both manifests and events are disabled no?